src/components/title/_fields/package-select/package-select-field.js

```diff
@@ -14,29 +13,22 @@ function PackageSelectField({ options }) {
     <div data-test-eholdings-package-select-field>
       <Field
         name="packageId"
-        component={Select}
+        component={Selection}
         label={<FormattedMessage id="ui-eholdings.label.package" />}
         validate={validate}
         onBlur={null} // preventing validation that is in onBlur
+        placeholder={<FormattedMessage id="ui-eholdings.title.chooseAPackage" />}
+        dataOptions={options.slice(0, 100)}
         required
-      >
-        {options.length ? (
-          <FormattedMessage id="ui-eholdings.title.chooseAPackage">
-            {option => (
-              <option value="" disabled>{option}</option>
-            )}
-          </FormattedMessage>
-        ) : (
-          <FormattedMessage id="ui-eholdings.search.loading">
-            {option => (
-              <option value="" disabled>{option}</option>
-            )}
-          </FormattedMessage>
-        )}
-        {options.filter(option => option.label !== '').map(({ disabled, label, value }) => (
```

Comment:
Please use `formatter` property of `Selection` component to support disabling options
https://ux.folio.org/storybook/?path=/story/selection--basic-usage